### ISSUE TITLE
Internal schema objects must be ignored by drift_dev schema dump

### ISSUE THREAD

**Issue by ycherniavskyi** [simolus3/moor#2217]
Dump the actual schema from the sqlite3 file mode of `schema dump` (added in #2208) doing a great job according to low-level schema manipulation. But it seems that it must skip [internal schema objects](https://www.sqlite.org/fileformat2.html#intschema) from a dump file. Because another way we are getting:
```
SqliteException(1): object name reserved for internal use: sqlite_sequence
  Causing statement: CREATE TABLE IF NOT EXISTS "sqlite_sequence" ("name" BLOB, "seq" BLOB);
```

**Comment by ycherniavskyi** [comment]
It seems the `drift_dev` already has [such functionality](https://github.com/simolus3/drift/blob/0e395bbef37b31d47abfa66e2efc81fed0129535/drift_dev/lib/src/services/schema/verifier_impl.dart#L86), but it isn't applied on the `schema dump` path.

**Comment by ycherniavskyi** [comment]
Here is my temporary workaround (I am not open MR because I think you could want to refactor this functionality and use already existing internal schema objects filtering mechanism):
```diff
diff --git a/drift_dev/lib/src/services/schema/sqlite_to_drift.dart b/drift_dev/lib/src/services/schema/sqlite_to_drift.dart
index 8e66de55..6b7bd001 100644
--- a/drift_dev/lib/src/services/schema/sqlite_to_drift.dart
+++ b/drift_dev/lib/src/services/schema/sqlite_to_drift.dart
@@ -18,7 +18,7 @@ Future<List<DriftElement>> extractDriftElementsFromDatabase(
     CommonDatabase database) async {
   // Put everything from sqlite_schema into a fake drift file, analyze it.
   final contents = database
-      .select('select * from sqlite_master')
+      .select('SELECT * FROM sqlite_master WHERE name NOT LIKE "sqlite_%"')
       .map((row) => row['sql'])
       .whereType<String>()
       .map((sql) => sql.endsWith(';') ? sql : '$sql;')
```

**Comment by simolus3** [comment]
Good catch, thanks for the report. I've fixed this in b5205689842e811540f42d48eecb091fcb1d53c2.
